src/panda_gui/panda_widgets.py

```diff
@@ -292,6 +315,21 @@ def updatePandaWidgets(self):
         self.tuningGUIUpdate.emit(ready_primitive)
         QApplication.restoreOverrideCursor()
 
+        # EUP State is checked and certain buttons are disabled accordingly
+        self.checkEUPState()
+
+        #We disabled frozen- or relaxed-buttons based on which state is active
```

Comment:
I think your pep8ify lost something. All inline comments should have a white space between `#` and text